tools/backup/backup.sh

```diff
@@ -244,19 +244,18 @@ function activate_gcp() {
 }
 
 function activate_aws() {
-  echo "Activating aws credentials before beginning"
-  mkdir -p /root/.aws/
-  cp /credentials/credentials ~/.aws/config
-
-  if [ $? -ne 0 ]; then
-    echo "Credentials failed; no way to copy to aws."
-    exit 1
-  fi
-
-  aws sts get-caller-identity
-  if [ $? -ne 0 ]; then
-    echo "Credentials failed; no way to copy to aws."
-    exit 1
+  local credentials="/credentials/credentials"
+  if [[ -f "${credentials}" ]]; then
+    echo "Activating aws credentials before beginning"
+    mkdir -p /root/.aws/
+    cp /credentials/credentials ~/.aws/config
+    aws sts get-caller-identity
+    if [ $? -ne 0 ]; then
+      echo "Credentials failed; no way to copy to aws."
+      exit 1
+    fi
```

Comment:
@alemairebe you've removed a 

```
if [ $? -ne 0 ]; then    echo "Credentials failed; no way to copy to aws."    exit 1  fi
```

after `cp /credentials/credentials ~/.aws/config`

Can you explain the reason ?

Reply:
it was remove before going to test if the file exists therefor, I had to let the script accept to fail copying credentials, I restored it. Thank you for bringing that up